internal/view/owner_extender.go

```diff
@@ -0,0 +1,233 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+	"github.com/derailed/k9s/internal/client"
+	"github.com/derailed/k9s/internal/ui"
+	"github.com/derailed/tcell/v2"
+	"github.com/derailed/tview"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const selectOwnerDialogKey = "owner"
+
+// OwnerExtender adds log actions to a given viewer.
+type OwnerExtender struct {
+	ResourceViewer
+}
+
+// NewOwnerExtender returns a new extender.
+func NewOwnerExtender(v ResourceViewer) ResourceViewer {
+	o := OwnerExtender{
+		ResourceViewer: v,
+	}
+	o.AddBindKeysFn(o.bindKeys)
+
+	return &o
+}
+
+// BindKeys injects new menu actions.
+func (o *OwnerExtender) bindKeys(aa ui.KeyActions) {
+	aa.Add(ui.KeyActions{
+		ui.KeyO: ui.NewKeyAction("Show Owner", o.ownerCmd(false), true),
+	})
+}
+
+func (o *OwnerExtender) ownerCmd(prev bool) func(evt *tcell.EventKey) *tcell.EventKey {
```

Comment:
Don't think we need to take in a bool arg here. This should just be a plain action handler.

Reply:
Looks like I copied the function signatures a bit too closely from another extender. Thanks for the hints, will all be cleaned up.